event-svc/src/tests/mod.rs

```diff
@@ -243,8 +243,8 @@ pub(crate) async fn generate_chained_events() -> Vec<ReconItem<EventId>> {
     let mut events: Vec<ReconItem<EventId>> = Vec::with_capacity(5);
 
     let signer = Box::new(signer().await);
-    let stream_id_1 = create_deterministic_stream_id(&[0x01]);
-    let stream_id_2 = create_deterministic_stream_id(&[0x02]);
+    let stream_id_1 = create_deterministic_stream_id_document(&[0x01]);
+    let stream_id_2 = create_deterministic_stream_id_model(&[0x02]);
```

Comment:
per explanation above, both of these should be model stream ids as they are the id of the stream that these new event belong to. Streams can only belong to model stream, not document streams.